modules/consensus/synchronize_test.go

```diff
@@ -171,7 +171,7 @@ func TestBlockHistory(t *testing.T) {
 	defer cst.closeCst()
 
 	// mine until we have enough blocks to test blockHistory
-	for cst.cs.Height() < MaxCatchUpBlocks {
+	for cst.cs.Height() < 50 {
```

Comment:
If you read the rest of the test, the data is input manually to represent what would happen in a request of 50.

And it has nothing to do with MaxCatchUpBlocks lol, I'm not sure why the constant was used in the first place.